### DISCUSSION TITLE
Regarding the GitHub page.Or show the project that uses the project on the official website

### DISCUSSION THREAD

**Discussion by admin8756** [saadeghi/daisyui#3162] [Ideas / feature requests]
For example, I use daisyui on my website.I think it is also possible to show people what kind of websites can be made using daisyui.

**Comment by admin8756** [comment]
> For example, I use daisyui on my website.I think it is also possible to show people what kind of websites can be made using daisyui.

This is a Chinese website.
My website is [leelaa.cn](leelaa.cn) and I think daisyui is so cool.It's very suitable for my aesthetic

**Comment by saadeghi** [comment]
Yeah It's a good idea 👍  
Nice website by the way.
